specs-i18n/search-wordpress-on-google.js

```diff
@@ -64,5 +64,8 @@ function doGoogleAdSearch( search_params ) {
 
 test_data.google_searches.forEach( function( search_params ) {
 	doGoogleAdSearch( search_params );
+	if ( process.env.CIRCLECI === 'true' ) {
+		sleep( 15000 );
```

Comment:
`sleep` here isn't defined. If you want to pause after each test, how about using a block like this in the test itself:

```
test.afterEach( function() {
	this.sleep(15000);
} );
```